integrations/helm/releasesync/utils.go

```diff
@@ -0,0 +1,33 @@
+package releasesync
+
+import (
+	ifv1 "github.com/weaveworks/flux/apis/helm.integrations.flux.weave.works/v1alpha"
+	"github.com/weaveworks/flux/integrations/helm/release"
+)
+
+func MappifyDeployInfo(releases map[string][]release.DeployInfo) map[string]map[string]int64 {
```

Comment:
What transformation is `Mappify`? Why is it necessary? Question applies to both `MappifyDeployInfo` and `MappifyReleaseFhrInfo`, which are almost identical.

Reply:
A)
Transforms a 

_map[string][]release.DeployInfo/ReleaseFhr structure_

       (a list of releases/custom resources keyed on namespace)

into

_map[string]map[string]int64 structure_

     (a map keyed on namespace, where each namespace key is associated with a map keyed on release/CR name)

It's purpose is to easily determine existence of a release or a CS.

B)
The two helper functions are similar, dealing with different types. One could have one function with more complex logic to cater for both cases. I find having two simpler ones preferable.